Dockerfile

```diff
@@ -52,10 +52,12 @@ RUN set -eux ; \
     intl \
     pcntl  \
     curl \
+    gd \
     exif ; \
     pecl install amqp ; \
     pecl install xdebug; \
     pecl install imagick ; \
```

Comment:
required to execute   
```
 docker-php-ext-configure gd   \
--with-webp \
--with-jpeg \
--with-xpm \
--with-freetype 

```
i you want add support for this files  types.
 and you need with apk install packages 


libpng-dev 
        jpeg-dev  
        freetype-dev  
        libwebp-dev 
        libzip-dev 


and

        libpng 
        libjpeg  
        freetype
        libwebp